src/shared/varlink-io.systemd.Network.c

```diff
@@ -45,6 +45,16 @@ static SD_VARLINK_DEFINE_METHOD(
                 SetPersistentStorage,
                 SD_VARLINK_DEFINE_INPUT(Ready, SD_VARLINK_BOOL, 0));
 
+static SD_VARLINK_DEFINE_METHOD(
+                SetLinkDown,
+                SD_VARLINK_DEFINE_INPUT(InterfaceIndex, SD_VARLINK_INT, SD_VARLINK_NULLABLE),
+                SD_VARLINK_DEFINE_INPUT(InterfaceName, SD_VARLINK_STRING, SD_VARLINK_NULLABLE));
```

Comment:
Please list the field for polkit by `VARLINK_DEFINE_POLKIT_INPUT`.

Also, for new methods, please add comments for each inputs. See other varlink-io.xyz.c.